src/domain/common/entities/DataElementRule.ts

```diff
@@ -3,14 +3,29 @@ import { DataElement } from "./DataElement";
 
 export type RuleType = "visible" | "disabled";
 
-export type DataElementRuleOptions = Record<RuleType, { dataElements: Code[]; condition: string }>;
+export type SingleConditionRule = {
+    type?: "single";
+    dataElements: Code[];
+    condition: string;
+};
+
+type MultipleConditionRule = {
+    type: "option";
+    conditions: SingleConditionRule[];
+};
+
+export type ConditionRule = SingleConditionRule | MultipleConditionRule;
+
+export type SingleDERuleOptions = Record<RuleType, ConditionRule>;
+
+export type DataElementRuleOptions = Record<RuleType, ConditionRule>;
```

Comment:
Do we need to keep both types: `SingleDERuleOptions` and `DataElementRuleOptions` they are identical, right?

Reply:
yes, you're right